src/firebase.ts

```diff
@@ -17,14 +17,21 @@ export const firebaseConfig = {
 };
 
 // Initialize Firebase
-
 const app = initializeApp(firebaseConfig);
 
-export const auth = getAuth();
-
 export const provider = new GoogleAuthProvider();
-//export const provider = new EmailAuthProvider();
-if (process.env.NODE_ENV === "development") {
-  connectAuthEmulator(auth, "http://127.0.0.1:9099");
+
+// for localdev only
+if (import.meta.env.MODE === "development") {
+  connectAuthEmulator(getAuth(), "http://127.0.0.1:9099");
+} else {
+  provider.setCustomParameters({
+    prompt: "select_account",
```

Comment:
what does this do?

Reply:
(I know this screenshot isn't perfectly relevant but if I had a second login it'd appear here okay :gun: )